ansible/roles/hyrax/meta/main.yml

```diff
@@ -11,3 +11,4 @@ dependencies:
   - { role: nodejs, nodejs_version: '6.x' }
   - { role: phantomjs, phantomjs_version: '2.1.1', when: project_app_env != 'production' }
   - { role: ruby, ruby_version: '2.3' }
+  - { role: local_files }
```

Comment:
The `local_files_dir` local files directory mechanism is utilised in more places than just the `sufia` and `hyrax` roles.  If you are wishing to populate local files from a remote archive then you will need to include the role in more places than those.

By my estimate, the local files directory is also used in these roles: `phantomjs`; `tls_cert`; `chroot-sftp`; and `fits`.

Reply:
From what I understand, those roles use files ('cached' files) which are already available in the `local_files_dir`, and as far as I can see, those roles are always called from the hyrax_site and sufia_site roles.

So, what I imagine is the proper course of action, is to put the `local_files` role before those roles as they are included in the hyrax_site and sufia_site roles. Or instead of explicitly including it in the super-roles, I can add it as a dependency for whichever roles which use `local_files_dir` (easily found out using `grep -r local_files_dir`).

Is my understanding of this correct?